src/um/audiosessiontypes.rs

```diff
@@ -27,9 +27,12 @@ pub const AUDCLNT_STREAMFLAGS_LOOPBACK: DWORD = 0x00020000;
 pub const AUDCLNT_STREAMFLAGS_EVENTCALLBACK: DWORD = 0x00040000;
 pub const AUDCLNT_STREAMFLAGS_NOPERSIST: DWORD = 0x00080000;
 pub const AUDCLNT_STREAMFLAGS_RATEADJUST: DWORD = 0x00100000;
+pub const AUDCLNT_STREAMFLAGS_PREVENT_LOOPBACK_CAPTURE: DWORD = 0x01000000;
+pub const AUDCLNT_STREAMFLAGS_SRC_DEFAULT_QUALITY: DWORD = 0x08000000;
 pub const AUDCLNT_SESSIONFLAGS_EXPIREWHENUNOWNED: DWORD = 0x10000000;
 pub const AUDCLNT_SESSIONFLAGS_DISPLAY_HIDE: DWORD = 0x20000000;
 pub const AUDCLNT_SESSIONFLAGS_DISPLAY_HIDEWHENEXPIRED: DWORD = 0x40000000;
+pub const AUDCLNT_STREAMFLAGS_AUTOCONVERTPCM: DWORD = 0x80000000;
```

Comment:
`AUDCLNT_STREAMFLAGS_AUTOCONVERTPCM` should be defined between `AUDCLNT_STREAMFLAGS_SRC_DEFAULT_QUALITY` and `AUDCLNT_SESSIONFLAGS_EXPIREWHENUNOWNED`.

Reply:
fixed